test/e2e/e2e_suite_test.go

```diff
@@ -191,6 +191,21 @@ var _ = Describe("operator-sdk", func() {
 			err = tc.Make("bundle-build", "BUNDLE_IMG="+bundleImage)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("checking the cluster type")
+			kubectx, err = tc.Kubectl.Command("config", "current-context")
+			Expect(err).Should(Succeed())
```

Comment:
Nit: We should avoid using `Expect(err).Should(Succeed())` to maintain readability.
```suggestion
			Expect(err).NotTo(HaveOccurred())
```
And to avoid repeating the issue https://github.com/operator-framework/operator-sdk/issues/3896